Aztec/Classes/Formatters/FontFormatter.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+import UIKit
+
+class FontFormatter: CharacterAttributeFormatter {
+
+    let elementType: Libxml2.StandardElementType
+    
+    let traits: UIFontDescriptorSymbolicTraits
+
+    init(elementType: Libxml2.StandardElementType, traits: UIFontDescriptorSymbolicTraits) {
+        self.elementType = elementType
+        self.traits = traits
+
+    }
+
+    func apply(to attributes: [String : Any]) -> [String: Any] {
+
+        var resultingAttributes = attributes
+        guard let font = attributes[NSFontAttributeName] as? UIFont else {
+            return attributes
+        }
+        let newFont = font.modifyTraits(traits, enable: true)
+        resultingAttributes[NSFontAttributeName] = newFont
+        
+        return resultingAttributes
+    }
+
+    func remove(from attributes:[String: Any]) -> [String: Any] {
```

Comment:
Add a space after `from attributes:`, and add unit tests.

- One that removes a present attribute.
- One that tries to remove a missing attribute.